doc/integrator/internationalization.rst

```diff
@@ -5,7 +5,8 @@
 Internationalization
 ====================
 
-In the file ``<package>.mk``, define the supported languages with (default):
+In the file ``<package>.mk``, define the supported languages with (default),
+in simple mode it's fixed to (en fr de it):
```

Comment:
"In the simple application mode, the list of languages is constant: en fr de it"